src/NumericalAlgorithms/DiscontinuousGalerkin/SimpleMortarData.hpp

```diff
@@ -21,6 +21,24 @@ namespace dg {
 template <typename TemporalId, typename LocalVars, typename RemoteVars>
 class SimpleMortarData {
  public:
+  SimpleMortarData() = default;
+  SimpleMortarData(const SimpleMortarData&) = default;
+  SimpleMortarData(SimpleMortarData&&) = default;
+  SimpleMortarData& operator=(const SimpleMortarData&) = default;
+  SimpleMortarData& operator=(SimpleMortarData&&) = default;
+  ~SimpleMortarData() = default;
+
+  /// The argument is ignored.  It exists for compatibility with
+  /// BoundaryHistory.
+  explicit SimpleMortarData(const size_t /*integration_order*/) noexcept {}
```

Comment:
Note for me: we might need this change in `Evolution/DiscontinuousGalerkin/MortarData.hpp`, but since in the new evolution code LTS boundary data is treated more distinctly from GTS mortar data, there might not be any generic code that needs these functions.